browser/net/brave_ad_block_tp_network_delegate_helper_unittest.cc

```diff
@@ -116,31 +116,40 @@ TEST_F(BraveAdBlockTPNetworkDelegateHelperTest, Blocking) {
 
 TEST_F(BraveAdBlockTPNetworkDelegateHelperTest, GetPolyfill) {
   GURL tab_origin("https://test.com");
+  GURL google_analytics_url(kGoogleAnalyticsPattern);
   GURL tag_manager_url(kGoogleTagManagerPattern);
   GURL tag_services_url(kGoogleTagServicesPattern);
   GURL normal_url("https://a.com");
   std::string out_url_spec;
+  // Shields up, block ads, google analytics should get polyfill
+  ASSERT_TRUE(GetPolyfillForAdBlock(true, false, tab_origin, google_analytics_url, &out_url_spec));
   // Shields up, block ads, tag manager should get polyfill
   ASSERT_TRUE(GetPolyfillForAdBlock(true, false, tab_origin, tag_manager_url, &out_url_spec));
   // Shields up, block ads, tag services should get polyfill
   ASSERT_TRUE(GetPolyfillForAdBlock(true, false, tab_origin, tag_services_url, &out_url_spec));
   // Shields up, block ads, normal URL should NOT get polyfill
   ASSERT_FALSE(GetPolyfillForAdBlock(true, false, tab_origin, normal_url, &out_url_spec));
 
+  // Shields up, allow ads, google analytics should get polyfill
+  ASSERT_FALSE(GetPolyfillForAdBlock(true, true, tab_origin, google_analytics_url, &out_url_spec));
   // Shields up, allow ads, tag manager should NOT get polyfill
   ASSERT_FALSE(GetPolyfillForAdBlock(true, true, tab_origin, tag_manager_url, &out_url_spec));
   // Shields up, allow ads, tag services should NOT get polyfill
   ASSERT_FALSE(GetPolyfillForAdBlock(true, true, tab_origin, tag_services_url, &out_url_spec));
   // Shields up, allow ads, normal URL should NOT get polyfill
   ASSERT_FALSE(GetPolyfillForAdBlock(true, true, tab_origin, normal_url, &out_url_spec));
 
+  // Shields down, allow ads, google analytics should get polyfill
+  ASSERT_FALSE(GetPolyfillForAdBlock(false, true, tab_origin, google_analytics_url, &out_url_spec));
```

Comment:
is this correct? shields down... we still want to polyfill? (or typo in comment)

Reply:
ah, yuck, typo.  Repushed and edd940b fixes